frontend/src/metabase/dashboard/components/Sidebar.tsx

```diff
@@ -46,20 +47,25 @@ export function Sidebar({
             >{t`Remove`}</Button>
           )}
           {onCancel && (
-            <ButtonDeprecated
-              small
-              borderless
+            <Button
+              variant="subtle"
+              color="text-medium"
               onClick={onCancel}
-            >{t`Cancel`}</ButtonDeprecated>
+              aria-label={t`Cancel`}
+            >{t`Cancel`}</Button>
           )}
           {onClose && (
-            <ButtonDeprecated
-              primary
-              small
-              className="ml-auto"
-              onClick={onClose}
-              disabled={closeIsDisabled}
-            >{t`Done`}</ButtonDeprecated>
+            <Tooltip label={closeTooltip} hidden={!closeTooltip}>
+              {/* without a div we will need hacks to make tooltip work */}
```

Comment:
Just out of curiosity, what kind of hacks? Does React Fragment not work here?

Reply:
Mantine turns off pointer events on a disabled button so that the tooltip can't receive a mouseenter or something like that. Therefore we need a real DOM element that is not messing with the mouse events.